haxe.YAML-tmLanguage

```diff
@@ -637,24 +637,30 @@ repository:
     name: meta.parameters.hx
     patterns:
     - include: '#parameter'
+    - include: '#punctuation-comma'
   parameter:
-    begin: (?<=\(|,)
-    end: (?=\)(?!\s*->)|,)
+    # name: "meta.parameter.hx"
+    begin: (?:(?<=\()|(?<=,))
+    end: (?=\)|,)
     patterns:
     - include: '#parameter-name'
     - include: '#parameter-type-hint'
     - include: '#parameter-assign'
-    - include: '#punctuation-comma'
     - include: '#global'
```

Comment:
after playing around with these changes, I found they seem to have no effect on the outcome. which means they are either not doing anything or we need more test cases to show they are needed.

Reply:
I think it marks comma as `source.hx meta.method.hx meta.parameters.hx punctuation.separator.comma.hx` in signature args instead of being part of `meta.parameters.hx`, but this doesn't matter much really

Reply:
I'll have to retest it, but I didn't see any changes between old or new versions of that section. 
e.g. the begin regex looks like a more complicated version of the original

Reply:
Maybe this is some leftover after i messed with these things so much time, lets left this comma change for next time, doesn't do anything useful anyway